bool valid
float64[3] translation
float64[4] orientation
builtin_interfaces/Time timestamp
